src/main/java/org/elasticsearch/indices/cluster/IndicesClusterStateService.java

```diff
@@ -308,7 +320,11 @@ private void applyNewIndices(final ClusterChangedEvent event) {
                 if (logger.isDebugEnabled()) {
                     logger.debug("[{}] creating index", indexMetaData.index());
                 }
-                indicesService.createIndex(indexMetaData.index(), indexMetaData.settings(), event.state().nodes().localNode().id());
+                try {
+                    indicesService.createIndex(indexMetaData.index(), indexMetaData.settings(), event.state().nodes().localNode().id());
+                } catch (Exception e) {
+                    logger.warn("failed to create index [{}]", e, indexMetaData.index());
```

Comment:
can we maintain the same logging format and have the index name at the start of the log?